pkg/debug/stop_debug.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package debug
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StopDebug(clusterNamespace, deploymentName string) {
+	// Get Kubernetes Client
+	_, clientset, _ := k8sutil.GetKubeClient()
+
+	if !strings.Contains(deploymentName, "-debug") {
+		deploymentName = deploymentName + "-debug"
+	}
+
+	debug_deployment, err := verifyDeploymentExists(clientset, clusterNamespace, deploymentName)
+	if err != nil {
+		fmt.Printf("Missing mon or osd deployment name %s. %v\n", deploymentName, err)
```

Comment:
If the error is that the debug deployment doesn't exist, let's continue and scale up the deployment. No need to fail when deleting something that doesn't exist.

Reply:
updated